ccd-definition/AuthorisationCaseEvent/CareSupervision/DEPRECATED.json

```diff
@@ -586,5 +586,16 @@
     "CaseEventID": "COMPLY_OTHERS",
     "UserRole": "caseworker-publiclaw-solicitor",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "hearingBookingDetails",
+    "AccessControl": [
```

Comment:
if single CRUD, AccessControl can be skipped, can be
`
"UserRoles": ["caseworker-publiclaw-courtadmin", "caseworker-publiclaw-gatekeeper", "caseworker-publiclaw-judiciary", "caseworker-publiclaw-magistrate", "caseworker-publiclaw-solicitor"],
        "CRUD": "R"
`